packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.test.tsx

```diff
@@ -704,6 +704,75 @@ describe('FocusZone', () => {
     setRTL(false);
   });
 
+  it('can focus correctly when receiving initial focus, bidirectionally', () => {
+    const component = ReactTestUtils.renderIntoDocument(
+      <div {...{ onFocusCapture: _onFocus }}>
+        <FocusZone>
+          <button className="a">a</button>
+          <button className="b">b</button>
+          <button className="c">c</button>
+          <button className="d">d</button>
+        </FocusZone>
+      </div>
+    );
+
+    const focusZone = ReactDOM.findDOMNode(component as React.ReactInstance)!.firstChild as Element;
+    const buttonA = focusZone.querySelector('.a') as HTMLElement;
+    const buttonB = focusZone.querySelector('.b') as HTMLElement;
+    const buttonC = focusZone.querySelector('.c') as HTMLElement;
+    const buttonD = focusZone.querySelector('.d') as HTMLElement;
+
+    // Set up a grid like so:
+    // A B
+    // C D
+    //
+    // We will focus B, and then down arrow, expecting D to be focused.
+
+    setupElement(buttonA, {
+      clientRect: {
+        top: 0,
+        bottom: 20,
+        left: 0,
+        right: 30
```

Comment:
Just curious, is this actually meant to overlap with Button B? (Button B's left is 20.)